src/actions/userActions.js

```diff
@@ -130,3 +135,41 @@ export const getProfile = () => dispatch => {
       dispatch({ type: LOGOUT_USER, payload: false });
     });
 };
+
+export const sendResetLink = (userDetails) => dispatch => {
+  toast.dismiss();
+  dispatch(sendResetLinkInitiated(true));
+  return axiosInstance
+    .post('/api/users/password_reset/', userDetails)
+    .then(response => {
+      dispatch(sendResetLinkSuccess(true));
+      toast.success(response.data.user.message, { autoClose: 3500, hideProgressBar: true });
+    })
+    .catch(() => {
+      dispatch(sendResetLinkError('Please enter a valid email'));
+      toast.error('Please enter a valid email', { autoClose: 3500, hideProgressBar: true });
+    });
+};
+
+export const resetPassword = (passwordDetails) => dispatch => {
+  toast.dismiss();
+  const token = localStorage.getItem('reset_password_token');
+  axios.defaults.headers.common.Authorization = `Token ${token}`;
+  axios
+    .put('https://ah-backend-targaryen-staging.herokuapp.com/api/users/password_update/', passwordDetails)
+    .then(response => {
+      dispatch(ResetPasswordSuccess(true));
+      toast.success(response.data.user.message, { autoClose: 3500, hideProgressBar: true });
+      localStorage.removeItem('reset_password_token');
+    })
+    .catch((error) => {
+      if (error.response) {
+        dispatch(ResetPasswordError(error.response.data.errors[0]));
```

Comment:
This will toast the error message and go ahead to toast the connection error all the time. Put the return statements back and return the axiosInstance.

Reply:
Oh yes,
I had forgotten that it will do that. 
Sorry for overlooking that.
I'm pushing the recommended changes now.